File: hw/arboleda/u3/HW28SingletonAndAbstractMethod/HW28AbstractMethod/src/main/java/ec/edu/espe/abstractmethod/controller/WinMenu.java
package ec.edu.espe.abstractmethod.controller;

/**
 *
 * @author Abner Arboleda, LogicLegion, DCCO-ESPE
 */
public class WinMenu extends Menu{

    public WinMenu() {
        super("Windows menu");
    }

    @Override
    public void paint() {
        System.out.println("In WinMenu: " + caption);
    }
    
}

File: hw/cañarte/u3/HW28SingletonAndAbstractFactory/Python/AbstractFactory/controller/WinMenu.py
import sys
import os
# Saray Cañarte, codecreators, DCCO - ESPE

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.Menu import Menu

class WinMenu(Menu):
    def display(self):
        print(f"I'm a WinMenu: {self.caption}")
